core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -168,9 +170,19 @@ private TabletsMetadata buildExtents(AccumuloClient client) {
             scanner.setRanges(ranges);
 
             configureColumns(scanner);
-            IteratorSetting iterSetting = new IteratorSetting(100, WholeRowIterator.class);
+            int iteratorPriority = 100;
+            IteratorSetting iterSetting =
```

Comment:
I think the WholeRowIterators priority should be set so that it comes last. It will encode entire rows into a single key value, so the filters will not see the individual columns when running after it.  So could add it after the for loop that adds filters with the highest prio.

Reply:
Addressed in c05549d